### PR TITLE
RFE#1594 : More plain english config words

### PR DESCRIPTION
**Pull request by devenbansod**
Replaces the existing use of 'script_names' like 'db_structure.php', 'sql.php' etc. as config options with more simple english words like 'structure', 'browse' etc.

Feedback welcome. Please suggest changes. 

Signed-off-by: Deven Bansod devenbansod.bits@gmail.com

### CODE REVIEW

**Comment by ibennetch** [comment]
I've begun reviewing this and so far everything looks good.

One note; we should also update `doc/config.rst` at least in the entries for
* `.. config:option:: $cfg['NavigationTreeDefaultTabTable']`
* `.. config:option:: $cfg['NavigationTreeDefaultTabTable2']`
* `.. config:option:: $cfg['NavigationTreeServer']`
* `.. config:option:: $cfg['NavigationTreeDatabase']`

I'll do some further tests and let you know if there are any problems.

**Comment by devenbansod** [comment]
Added the docs.

**Comment by ibennetch** [comment]
I believe we can improve the look of the setup form, which I think you were asking about by email, by making this change to `libraries/config.values.php`

<pre>
-    'structure', // fields list
+    'structure' => __('Structure'), // fields list
</pre>

And so on.

File: libraries/config.values.php [hunk 1]
```diff
@@ -36,19 +36,19 @@
...
-    'tbl_structure.php', // fields list
-    'tbl_sql.php',       // SQL form
-    'tbl_select.php',    // search page
-    'tbl_change.php',    // insert row page
-    'sql.php'            // browse page
+    'structure', // fields list
+    'sql',       // SQL form
+    'search',    // search page
+    'insert',    // insert row page
+    'sql'        // browse page
```
Comment (ibennetch) [inline libraries/config.values.php]: Do you mean `'browse'` here?

File: libraries/config.values.php [hunk 2]
```diff
@@ -36,19 +36,19 @@
...
```
Comment (ibennetch) [inline libraries/config.values.php]: And `'browse'` here as well?

**Comment by ibennetch** [comment]
Nice work. If you can fix the three items listed above -- two 'sql' lines where I think you meant 'browse' and adding the `=>` syntax in `config.values.php`, then squash it all and I'll do one more quick review before merging.

**Comment by devenbansod** [comment]
Thanks for pointing that out! I don't know how could I miss it!

Anyways, updated and squashed.

**Comment by ibennetch** [comment]
Nice job Deven.

**Comment by devenbansod** [comment]
Thanks.

**Comment by devenbansod** [comment]
Please see #1729

**Comment by devenbansod** [comment]
Also, please see #1739

**Pull request merged**